build/htmllint.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/*!
+ * Script to run vnu-jar if Java is available.
+ * Copyright 2017 The Bootstrap Authors
+ * Copyright 2017 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+'use strict'
+
+const childProcess = require('child_process')
+const vnu = require('vnu-jar')
+
+childProcess.exec('java -version', function (error) {
+  if (error) {
+    console.error('Skipping HTML lint test; Java is missing.')
```

Comment:
If you return directly after that console.error you should throw an error instead

Reply:
We don't want to fail here; if Java isn't installed we are just skipping vnu-jar.